packages/legend-application-query/src/stores/QueryEditorStore.ts

```diff
@@ -565,6 +570,11 @@ export class ServiceQueryEditorStore extends QueryEditorStore {
     }
     // leverage initialization of query builder state to ensure we handle unsupported queries
     this.queryBuilderState.initialize(service.execution.func);
+    await Promise.all([
```

Comment:
See my comment in `QueryBuilderChangeDetectionState`, we should only need to do:
`await queryBuilderState.changeDetectionState.start();` here